stories/Button.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
I left all these in to reference.. can remove?

Reply:
can we comment somewhere to say that they are here for reference? rename the folder?
agree that they're good to have, just threw me at first, wondering why there were stories here, and next to components